src/views/Header/HeaderTitle.js

```diff
@@ -8,7 +8,9 @@ const HeaderTitle = ({ style, ...rest }) => (
     numberOfLines={1}
     {...rest}
     style={[styles.title, style]}
+    accessible={true}
     accessibilityTraits="header"
+    accessibilityLabel={'nav_header_title'}
```

Comment:
Not sure if we should hardcode the accessibility label to this. Shouldn't it read the title of the screen?

Reply:
Without accessibilityLabel Apium  can't read Header Title. It can be const because on one screen can be only one Header with one HeaderTitle.

Reply:
For Appium it works, but how about screen readers? For people who use accessibility tools, I think this will provide a bad experience.

Reply:
Screen reader will read header title. Why is it bad experience?

Reply:
@satya164 I've fixed all issues for merge. Please review. It's really very necessary fix for autotesting. And it'll not provide a bad experience for using accessibility tools.

Reply:
@satya164 could you please merge?

Reply:
> Screen reader will read header title. Why is it bad experience?

Screen reader needs to read the actual title in the header, not the text `header_title`